src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap);
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0 auto;
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.container {
```

Comment:
Why do you need this container ?
Maybe, you could add these styles in `header` selector ?